webpack.config.js

```diff
@@ -42,6 +42,8 @@ config.plugins.push( new CleanWebpackPlugin( {
 
 } ) );
 
+// config.entry.fontawesome = resolve( './src/scss/fontawesome.scss' );
```

Comment:
No, since the LifterLMS core won't be using FA 6 (yet) we can leave it out. We can uncomment it later when we decide to transition from FA 4 -> 6 in the core. For now I'll just leave it commented out.

Reply:
ok fine